certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json

```diff
@@ -437,19 +450,23 @@
                     "properties": {
                       "file_name": {
                         "type": "string",
-                        "description": "file_name"
+                        "description": "file_name",
+                        "title": "file name"
                       },
                       "diseases_found": {
                         "type": "string",
-                        "description": "diseases_found"
+                        "description": "diseases_found",
+                        "title": "diseases found"
                       },
                       "cpt_code": {
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_code_description",
```

Comment:
@gayathri35 All the description values of properties should be free from grammatical and spell errors. please provide valid description values for all the properties.

Reply:
Hi@sriyen-msft,
Sorry for the mistake, I have corrected it please do check once,
 },
                      "cpt_description": {
                        "type": "string",
                        "description": "cpt_code_description",
                        "title": "cpt code description"
                      },

Reply:
@gayathri35 Title and Description values are different. I have provided feedback for description values of all the properties in the swagger. 

Ex.: Line number 468: `"description": "cpt_code_description"` - This is not allowed it has grammatical and spell errors. Please fix all other description values of properties in the swagger.

Reply:
@gayathri35 Please resolve this comment to get your PR approved and merged to move forward in the certificaiton.

Reply:
Hi Srikanth,
I have changed the description comment. Please do check once, Sorry for the mistake.
"cpt_desc": {
                        "type": "string",
                        "description": "cpt_description",
                        "title": "cpt desc"
                      },